Fix KcAdmV2EditCLITest.testEditNonExistentResource assertion when client is not found

### Before reporting an issue

- [x] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/cli

### Describe the bug

`KcAdmV2EditCLITest.testEditNonExistentResource` assertion has changed on the main branch while we were reviewing https://github.com/keycloak/keycloak/pull/48242. Sorry, I waited with rebase for the requests for changes.

### Version

current main only

### Regression

- [ ] The issue is a regression

### Expected behavior

Pass.

### Actual behavior

```
[ERROR] Failures: 
[ERROR]   KcAdmV2EditCLITest.testEditNonExistentResource:94 
Expected: is "Could not find client"
     but: was "Cannot find the specified client"
[INFO] 
[ERROR] Tests run: 7, Failures: 1, Errors: 0, Skipped: 0
```

### How to Reproduce?

`mvn test -pl rest/admin-v2/tests -Dtest="KcAdmV2EditCLITest"`

### Anything else?

_No response_

--- a/rest/admin-v2/tests/src/test/java/org/keycloak/tests/admin/cli/v2/KcAdmV2EditCLITest.java
+++ b/rest/admin-v2/tests/src/test/java/org/keycloak/tests/admin/cli/v2/KcAdmV2EditCLITest.java
@@ -91,7 +91,7 @@ public class KcAdmV2EditCLITest extends AbstractKcAdmV2CLITest {
         CommandResult result = kcAdmV2Cmd("client", "edit", "non-existent-id");
 
         assertThat("edit non-existent should fail", result.exitCode(), is(not(0)));
-        assertThat(result.err().trim(), is("Could not find client"));
+        assertThat(result.err().trim(), is("Cannot find the specified client"));
     }
 
     @Test

